### ISSUE TITLE
Unable to run `./nodebb setup` to launch interactive installer

### ISSUE THREAD

**Issue by julianlam** [NodeBB/NodeBB#10323]
After #9850 was merged, it was discovered that `./nodebb setup` was no longer acting properly if no arguments were passed in. It should kickstart the interactive cli installer. Instead, it fell into the env var checking logic, and returned an error that required env vars were missing.

#10312 closes this issue.

**Comment by julianlam** [comment]
Backported to v1.19.x
